src/app/ipv/middleware.js

```diff
@@ -224,7 +193,6 @@ function isValidPage(pageId) {
 }
 
 function handleAppStoreRedirect(req, res, next) {
-  // PYIC-4816 - consider whether we should override this with the current request's sniffed device type.
```

Comment:
Was this discussed with anyone?

Reply:
Would like to go through this with you because I don't see the benefit